src/Entities/Category.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Fsbe\Entities;
+
+class Category implements \JsonSerializable
+{
+    private int $id;
+    private string $name;
+    private int $products;
+
+    /**
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
+     */
+    public function setId(int $id): void
```

Comment:
not sure if allowing overriding of id (primary key assigned by db) is a good idea. potential for removal

Reply:
same for all of the setters in this object, if they arent needed or being used then prob shouldnt be there